### DISCUSSION TITLE
Any source to find how to convert float/double to string?

### DISCUSSION THREAD

**Discussion by rempas** [MrSmith33/vox#31] [Q&A] [answered]
Do you know any source that will either explain me how to convert a float/double to string or even better provide the complete code for me?

**Comment by MrSmith33** [comment, marked as answer]
Conversion of floats to string is pretty complicated topic. I'm searching for compact and easy to port sources myself, but the smaller the method, the more flaws it has. [Ryu](https://github.com/ulfjack/ryu) one is the latest research (see a list of implementations there). Of course it matters only if you want precise conversion between float and string.

I've only looked at a couple ryu implementations, but the one written in [Go](https://github.com/cespare/ryu) is pretty compact and readable, I think.

**Reply by rempas** [reply]
Thanks, I also heard that float conversion is complicated but we don't have a system library and someone must create it so here I am...
